src/platform/esp32/main-esp32.cpp

```diff
@@ -139,12 +141,19 @@ void esp32Setup()
 #if !MESHTASTIC_EXCLUDE_BLUETOOTH
     String BLEOTA = BleOta::getOtaAppVersion();
     if (BLEOTA.isEmpty()) {
-        LOG_INFO("No OTA firmware available");
+        LOG_INFO("No BLE OTA firmware available");
     } else {
-        LOG_INFO("OTA firmware version %s", BLEOTA.c_str());
+        LOG_INFO("BLE OTA firmware version %s", BLEOTA.c_str());
     }
-#else
-    LOG_INFO("No OTA firmware available");
+#endif
+#if !MESHTASTIC_EXCLUDE_WIFI
+    String version = WiFiOTA::getVersion();
+    if (version.isEmpty()) {
+        LOG_INFO("No WiFi OTA firmware available");
+    } else {
+        LOG_INFO("WiFi OTA firmware version %s", version.c_str());
+    }
```

Comment:
wouldn't that print that a wifi AND ble updater was available, since the wifi check only looks for the version number? BLEOTA has a version number too :-)